Team18Project/app/src/main/java/com/example/team18project/StatsFragment.java

```diff
@@ -18,29 +19,26 @@ public class StatsFragment extends Fragment {
     // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
     private static final String ARG_PARAM1 = "param1";
     private static final String ARG_PARAM2 = "param2";
-
+    private static final String ARG_PARAM3 = "param3";
     // TODO: Rename and change types of parameters
-    private String mParam1;
-    private String mParam2;
-
-    public StatsFragment() {
-        // Required empty public constructor
-    }
+    private int totalScore;
+    private int high_score;
+    private int low_score;
 
     /**
      * Use this factory method to create a new instance of
      * this fragment using the provided parameters.
      *
-     * @param param1 Parameter 1.
-     * @param param2 Parameter 2.
+     * @param totalScore Score of all player QR codes added together
      * @return A new instance of fragment StatsFragment.
      */
     // TODO: Rename and change types and number of parameters
-    public static StatsFragment newInstance(String param1, String param2) {
+    public static StatsFragment newInstance(int totalScore,int highest_score,int lowest_score) {
```

Comment:
newInstance() should probably take the player object as an argument and bundle that instead, since other parts of the stats screen will need other information

Reply:
can't bundle objects